<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QFrame" name="cmdFrame" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>951</width>
      <height>46</height>
     </rect>
    </property>
    <property name="styleSheet" >
     <string>QFrame {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.461 rgba(232, 232, 232, 255), stop:0.463 rgba(202, 202, 202, 255), stop:1 rgba(237, 237, 237, 255))
}</string>
    </property>
    <property name="frameShape" >
     <enum>QFrame::NoFrame</enum>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="styleSheet" >
        <string>color: black
</string>
       </property>
       <property name="text" >
        <string>Title</string>
       </property>
       <property name="buddy" >
        <cstring>titleEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="titleEdit" >
       <property name="palette" >
        <palette>
         <active>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="styleSheet" >
        <string>background-color: white</string>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="pictureBorderCheckbox" >
       <property name="styleSheet" >
        <string>color: black;</string>
       </property>
       <property name="text" >
        <string>Picture border</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pictureBackgroundButton" >
       <property name="styleSheet" >
        <string>background-color : transparent; color:black</string>
       </property>
       <property name="text" >
        <string>Picture background</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadButton" >
       <property name="palette" >
        <palette>
         <active>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button" >
           <brush brushstyle="SolidPattern" >
            <color alpha="0" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button" >
           <brush brushstyle="SolidPattern" >
            <color alpha="0" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button" >
           <brush brushstyle="SolidPattern" >
            <color alpha="0" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="styleSheet" >
        <string>background-color : transparent; color:black
</string>
       </property>
       <property name="text" >
        <string>Load...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton" >
       <property name="palette" >
        <palette>
         <active>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button" >
           <brush brushstyle="SolidPattern" >
            <color alpha="0" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button" >
           <brush brushstyle="SolidPattern" >
            <color alpha="0" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button" >
           <brush brushstyle="SolidPattern" >
            <color alpha="0" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="styleSheet" >
        <string>background-color: transparent;
color:black;</string>
       </property>
       <property name="text" >
        <string>Save...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addPictures" >
       <property name="styleSheet" >
        <string>background-color: transparent;
color:black;</string>
       </property>
       <property name="text" >
        <string>Add Pictures</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pngButton" >
       <property name="palette" >
        <palette>
         <active>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button" >
           <brush brushstyle="SolidPattern" >
            <color alpha="0" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button" >
           <brush brushstyle="SolidPattern" >
            <color alpha="0" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button" >
           <brush brushstyle="SolidPattern" >
            <color alpha="0" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="styleSheet" >
        <string>background-color: transparent;
color:black;</string>
       </property>
       <property name="text" >
        <string>Render PNG</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton" >
       <property name="styleSheet" >
        <string>background-color: transparent;
color:black;</string>
       </property>
       <property name="text" >
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" native="1" name="Widget" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>40</y>
      <width>951</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet" >
     <string>background-color: transparent;
color:black;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>952</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionLoad" />
    <addaction name="actionSave" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuImage_effects" >
    <property name="title" >
     <string>Image effects</string>
    </property>
    <addaction name="actionBlack_and_White" />
    <addaction name="actionNVG" />
    <addaction name="actionInvert_colors" />
    <addaction name="actionHorizontal_flip" />
    <addaction name="actionVertical_flip" />
    <addaction name="actionLuminosity" />
   </widget>
   <widget class="QMenu" name="menuImage" >
    <property name="title" >
     <string>Image</string>
    </property>
    <addaction name="actionRotate_90_left" />
    <addaction name="actionRotate_90_right" />
    <addaction name="actionSet_as_background" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation" />
   </widget>
   <addaction name="menuFichier" />
   <addaction name="menuImage" />
   <addaction name="menuImage_effects" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionBlack_and_White" >
   <property name="text" >
    <string>Black and White (W)</string>
   </property>
  </action>
  <action name="actionInvert_colors" >
   <property name="text" >
    <string>Invert colors         (I)</string>
   </property>
  </action>
  <action name="actionLoad" >
   <property name="text" >
    <string>Load...</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="text" >
    <string>Save...</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
  <action name="actionHorizontal_flip_H" >
   <property name="text" >
    <string>Horizontal flip (H)</string>
   </property>
  </action>
  <action name="actionVertical_flip" >
   <property name="text" >
    <string>Vertical flip           (V)</string>
   </property>
  </action>
  <action name="actionHorizontal_flip" >
   <property name="text" >
    <string>Horizontal flip      (H)</string>
   </property>
  </action>
  <action name="actionRotate_90_left" >
   <property name="icon" >
    <iconset>data/rotate_left.png</iconset>
   </property>
   <property name="text" >
    <string>Rotate 90° left       (L)</string>
   </property>
  </action>
  <action name="actionRotate_90_right" >
   <property name="icon" >
    <iconset>data/rotate_right.png</iconset>
   </property>
   <property name="text" >
    <string>Rotate 90° right    (R)</string>
   </property>
  </action>
  <action name="actionSet_as_background" >
   <property name="text" >
    <string>Set as background</string>
   </property>
  </action>
  <action name="actionDocumentation" >
   <property name="text" >
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionNVG" >
   <property name="text" >
    <string>Level of gray (NVG)  (B)</string>
   </property>
  </action>
  <action name="actionLuminosity" >
   <property name="text" >
    <string>Luminosity</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
